.github/workflows/windows.yml

```diff
@@ -339,17 +328,26 @@ jobs:
         if: ${{ matrix.tflags != 'skipall' && matrix.tflags != 'skiprun' }}
         timeout-minutes: 5
         run: |
+          /usr/bin/sed -i 's/^CheckSpace/#CheckSpace/' /etc/pacman.conf
```

Comment:
FYI, you don't need the sed when using setup-msys2, it already does it.  For instance, https://github.com/curl/curl/actions/runs/13883646713/job/38845448400#step:2:21

Reply:
Ah, thanks, I got it now. With setup-msys2 in all jobs now, we can drop all 3 of these.

Reply:
PR → #16739